src/ophys_etl/decrosstalk/decrosstalk_types.py

```diff
@@ -0,0 +1,334 @@
+from typing import List, Tuple
+import numpy as np
+
+
+class BasicDictWrapper(object):
+
+    def __contains__(self, key) -> bool:
+        if key in self._data:
```

Comment:
Can be simplified to `return key in self._data`

Reply:
done